getting-started/scaling-out.md

```diff
@@ -0,0 +1,179 @@
+[`timescaledb_information.data_node`][timescaledb_information-data_node] view.
+
+## Working with Distributed Hypertables
+
+As previously mentioned, distributed hypertables are similar to normal
+hypertables with the difference that distributed hypertables will
+partition the data and store it on remote data nodes. This means that
+to create a distributed hypertable, you need to have added at least one data node
+before starting.
+
+### Creating Distributed Hypertables
+
+As with normal hypertables, start by creating a regular
+table to hold the data, then transform it into a distributed
+hypertable. In contrast to when creating a normal hypertable, we use
+the function
+[`create_distributed_hypertable`][create_distributed_hypertable].
+
+```sql
+CREATE TABLE conditions (
+  time        TIMESTAMPTZ       NOT NULL,
+  location    TEXT              NOT NULL,
+  temperature DOUBLE PRECISION  NULL,
+  humidity    DOUBLE PRECISION  NULL
+);
+
+SELECT create_distributed_hypertable('conditions', 'time', 'location');
+```
+
+>:WARNING: If there are no data nodes available, you will get an error
+>when executing `create_distributed_hypertable` and the distributed
+>hypertable will not be created.
+
+You can now insert data into the distributed hypertable and
+it will automatically be partitioned on the available data nodes
+by using the provided space partition. You
+can find more information for how work with data in hypertables in the
+section [Creating Hypertables][creating-hypertables].
+
+### Changing the Number of Data Nodes for a Distributed Hypertable
+
+By default, distributed hypertables will use all available data nodes,
+but individual data nodes can be added and removed from already
+created distributed hypertables using
+[`attach_data_node`][attach_data_node] and
+[`detach_data_node`][detach_data_node].
+
+For example, if you add a new data node to the distributed database, 
+it will not automatically be
+added to existing distributed hypertables, so it is necssary to attach
+it explicitly.
+
+```sql
+SELECT add_data_node('node3', host => 'dn3.example.com',
+  password=>'<remote_password>', bootstrap_user=>'<superuser>',
+  bootstrap_password=>'<superuser_password>');
+SELECT attach_data_node('node3', hypertable => 'conditions');
+```
+
+Once the data node has been attached, any new chunks will also be
+distributed to the hypertable on the new data node.
+
+In a similar way, if you want to remove a data node from a distributed
+hypertable, you can use [`detach_data_node`][detach_data_node].
+
+```sql
+SELECT detach_data_node('node1', hypertable => 'conditions');
+```
+
+If a data node is storing data for a hypertable,
+then you will get an error.
+
+```sql
+SELECT detach_data_node('node1', hypertable => 'conditions');
+ERROR:  detaching data node "node1" would mean a data-loss for hypertable "conditions" since data node has the only data replica
+HINT:  Ensure the data node "node1" has no non-replicated data before detaching it.
```

Comment:
Again, this alludes to replication being a feature we're supporting right now. Maybe we should just say:

```suggestion
HINT:  Ensure the data node "node1" has no data before detaching it.
```

Reply:
@erimatnor Current text is printed by the current implementation. Has nothing to do with the doc. Needs to be fixed in the source code.